bazel_configure.py

```diff
@@ -413,7 +413,7 @@ def main():
   env_map['AUTOMAKE'] = discover_tool('automake', 'Automake', 'AUTOMAKE', '1.9.6')
   env_map['AUTOCONF'] = discover_tool('autoconf', 'Autoconf', 'AUTOCONF', '2.6.3')
   env_map['MAKE'] = discover_tool('make', 'Make', 'MAKE', '3.81')
-  env_map['PYTHON3'] = discover_tool('python3', 'Python3', 'PYTHON3', '3.4')
+  env_map['PYTHON3'] = discover_tool('python3', 'Python3', 'PYTHON3', '3.5')
 
```

Comment:
this should probably check that `python -m venv` returns non-zero (the version will be same as the python interpreter)

Reply:
Should this be bumped up to 3.6?

Reply:
Do you have a suggestion of how best to do this? Seems the `discover_tool` method wouldn't work for what you describe, right?

Reply:
I added the test function. Let me know if it looks good.